docs/digger/installation/install-jenkins.asciidoc

```diff
@@ -0,0 +1,9 @@
+=== Install Jenkins
+
+The installer will perform the following tasks in order to setup and configure Jenkins:
+
+* Create the Jenkins service on OpenShift. It will automatically create the required routes, services and pods, and wait for Jenkins to be up and running. It will also create an admin user by default.
+* Then it will use the https://wiki.jenkins.io/display/JENKINS/Jenkins+CLI[Jenkins CLI] tool to configure Jenkins. However, this tool requires users to upload their public keys to Jenkins. The installer will stop and print out instructions of how to do that.
+* Once the key is uploaded, the installer will continue on. It will update configurations of Jenkins, and install the required plugins.
+
+You can use the tag `deploy-jenkins` to only perform the task to setup Jenkins.
```

Comment:
same as my comment above. The Ansible installer should print this information out. No need to ask the admin user to find them out on OpenShift.